corehq/apps/reports/views.py

```diff
@@ -1348,6 +1340,7 @@ def form_to_json(form):
     ])
 
 
+@location_safe
 class CaseAttachmentsView(CaseDetailsView):
```

Comment:
yeah, the dispatch function calls the `CaseDetailsView` dispatch function which has that logic in it